docs/src/components/examples/VirtualTable.js

```diff
@@ -36,36 +39,93 @@ const users = range(1000)
   })
 
 export default class VirtualTable extends React.PureComponent {
+  state = {
+    scrollToIndex: '',
```

Comment:
Since `scrollToIndex` and `scrollOffset` are not strings, but numbers, default values here should be `null`, not `''`.

Reply:
Nvm, fixed